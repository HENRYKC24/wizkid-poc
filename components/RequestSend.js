import { StyleSheet, Text, TouchableOpacity, View } from "react-native";
import React from "react";

const RequestSend = () => {
  return (
    <View style={styles.container}>
      <TouchableOpacity>
        <Text style={styles.item}>Request</Text>
      </TouchableOpacity>
      <TouchableOpacity>
        <Text style={styles.item}>Send</Text>
      </TouchableOpacity>
    </View>
  );
};

export default RequestSend;

const styles = StyleSheet.create({
  container: {
    flexDirection: "row",
    marginTop: 25,
    justifyContent: "center",
  },
  item: {
    marginHorizontal: 5,
    backgroundColor: "#e0e0e0",
    borderRadius: 20,
    width: 132,
    paddingVertical: 6,
    textAlign: "center",
    color: "#888",
    fontWeight: "700",
  },
});
