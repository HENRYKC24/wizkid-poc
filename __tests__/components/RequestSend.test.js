import React from "react";
import renderer from "react-test-renderer";

import RequestSend from "../../components/RequestSend";

describe("<RequestSend />", () => {
  it("has 2 children", () => {
    const tree = renderer.create(<RequestSend />).toJSON();
    expect(tree.children.length).toBe(2);
  });

  it("renders correctly", () => {
    const tree = renderer.create(<RequestSend />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
