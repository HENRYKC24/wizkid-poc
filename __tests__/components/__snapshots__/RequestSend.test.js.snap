// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RequestSend /> renders correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "justifyContent": "center",
      "marginTop": 25,
    }
  }
>
  <View
    accessible={true}
    collapsable={false}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text
      style={
        Object {
          "backgroundColor": "#e0e0e0",
          "borderRadius": 20,
          "color": "#888",
          "fontWeight": "700",
          "marginHorizontal": 5,
          "paddingVertical": 6,
          "textAlign": "center",
          "width": 132,
        }
      }
    >
      Request
    </Text>
  </View>
  <View
    accessible={true}
    collapsable={false}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text
      style={
        Object {
          "backgroundColor": "#e0e0e0",
          "borderRadius": 20,
          "color": "#888",
          "fontWeight": "700",
          "marginHorizontal": 5,
          "paddingVertical": 6,
          "textAlign": "center",
          "width": 132,
        }
      }
    >
      Send
    </Text>
  </View>
</View>
`;
